Makefile

```diff
@@ -13,11 +13,11 @@ endif
 
 ifneq ("$(wildcard /usr/share/selinux/devel/include/*/docker.if)","")
 export M4PARAM += -D has_docker
-else
-ifneq ($(DISTRO),rhel6)
+else ifneq ("$(wildcard /usr/share/selinux/devel/include/*/container.if)","")
+export M4PARAM += -D has_docker
+else ifneq ($(DISTRO),rhel6)
 $(error *** Interface docker.if not present, cannot continue ***)
```

Comment:
Nitpick we could print `*** Interface docker.if or container.if not present, cannot continue ***`